composer.json

```diff
@@ -26,7 +26,7 @@
         "phpunit/phpunit": "~4.0",
         "psy/psysh": "~0.5.1",
         "satooshi/php-coveralls": "~0.6.1",
-        "symfony/var-dumper": "2.7.*"
+        "symfony/var-dumper": "~2.7|3.0.*"
     },
     "autoload": {
         "psr-4": {
```

Comment:
I am not sure that I follow the question. We had it locked to 2.7.*, but Laravel now supports >=2.7 & >=3, so this makes it follow Laravel.